packages/engine-server/src/DendronEngineV3.ts

```diff
@@ -143,14 +143,15 @@ export class DendronEngineV3 extends EngineV3Base implements DEngine {
       vaults: ConfigUtils.getVaults(config),
       noteStore: new NoteStore(
         fileStore,
-        new NoteMetadataStore(),
+        new FuseMetadataStore(),
```

Comment:
its a problem right now that the `FuseMetadataStore`  is initialized twice. the SQLite metadata store needs to pull down binaries and initialize tables on the initial load. either we 
- introduce an `init` method to the metadatastore/queryEngine 
- initialize it/inject it outside of DendronEngine 
- combine metadatastore and queryEngine

Reply:
Sorry for the confusion - this `create` method was just scratch work - I ended up moving the `create()` method intended for use into `EngineV3Factory.ts`, where a single FuseMetadatStore is used. 

But all the code is still just prototyping.